torax/_src/physics/tests/psi_calculations_test.py

```diff
@@ -190,6 +192,58 @@ def test_calc_Wpol(self):
     # our circular geometry, but approximates it at low inverse aspect ratio.
     np.testing.assert_allclose(calculated_Wpol, expected_Wpol, rtol=1e-3)
 
+  # pylint: enable=invalid-name
+
+  def test_calc_j_tor_from_j_parallel(self, step_geo_and_j_profiles_from_IMAS):
```

Comment:
This could potentially be moved to test_utils/torax_refs. However, it's a bit of a different setup to the code that's already there, as we're interested in j_toroidal vs j_parallel rather than the psi, psidot, q, s profiles.